kerasfuse/models/attention_unet.py

```diff
@@ -0,0 +1,91 @@
+import tensorflow as tf
+
+
+def attention_block(input1, input2):
+    """
+    Attention block used in the attention U-Net model.
+
+    Args:
+        input1 (Tensor): First input tensor.
+        input2 (Tensor): Second input tensor.
+
+    Returns:
+        output (Tensor): Output tensor after applying attention.
+    """
+    g = tf.keras.layers.Conv2D(filters=input1.shape[-1], kernel_size=1)(input1)
+    x = tf.keras.layers.Conv2D(filters=input2.shape[-1], kernel_size=1)(input2)
+    psi = tf.keras.layers.Add()([g, x])
+    psi = tf.keras.layers.Activation("relu")(psi)
+    psi = tf.keras.layer.Conv2D(filters=1, kernel_size=1)(psi)
```

Comment:
this must be `psi = tf.keras.layers.Conv2D(filters=1, kernel_size=1)(psi)`

Reply:
Have made the necessary changes, must look good now.